code/ship/shiphit.cpp

```diff
@@ -652,10 +674,9 @@ float do_subobj_hit_stuff(object *ship_objp, object *other_obj, vec3d *hitpos, i
 					min_index = i;
 				}
 			}
+			Assert(min_index != -1);
```

Comment:
I only touched it to move it -- because it's only needed within that if() block.  The only reason it should ever be triggered is because of a programming error.  Adding a description would only draw attention to it, in my opinion.  If it's really a problem then I'll simply take it out.

Reply:
not enough of a problem to delay approval, I just don't understand why a 5 min fix isn't worth the extra clarity

Reply:
That's where we disagree.  For this particular `Assert`, adding an unnecessary description impedes clarity of the rest of the code.  That isn't the case for many other `Assert`s.